upload_controller_service/api/main.py

```diff
@@ -56,3 +57,30 @@ async def get_presigned_post(
         ) from file_info_not_found_error
 
     return {"presigned_post": url}
+
+
+@app.get("/confirm_upload/{file_id}", summary="confirm_upload")
+async def confirm_upload(
+    file_id: str,
+    config: Config = Depends(get_config),
+):
+    """
+    Requesting a confirmation of the upload of a specific file using the file id.
+    Returns 200, if the file exists in the inbox, 404 if not
+    """
+
+    # call core functionality
+    try:
+        check_uploaded_file(file_id=file_id, config=config)
```

Comment:
This function is nice, however, shouldn't we also track that the file has now been declared as "uploaded"?

Reply:
Will be done in another PR.

Reply:
https://jira.verbis.dkfz.de/browse/GDEV-478